src/main/java/net/atos/zac/app/formulieren/model/RESTFormulierDefinitie.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package net.atos.zac.app.formulieren.model;
+
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class RESTFormulierDefinitie {
+
+    public Long id;
+
+    public String systeemnaam;
+
+    public String naam;
+
+    public String beschrijving;
+
+    public String uitleg;
+
+    public ZonedDateTime creatiedatum;
+
+    public ZonedDateTime wijzigingsdatum;
+
+    public List<RESTFormulierVeldDefinitie> veldDefinities = new ArrayList<>();
```

Comment:
Hier initialiseer je de List. Maar eigenlijk doen we dat nooit. Ik zou het hier dan ook niet doen,